common/src/main/java/org/apache/gravitino/dto/requests/PolicyCreateRequest.java

```diff
@@ -0,0 +1,170 @@
+import org.apache.gravitino.policy.Policy;
+import org.apache.gravitino.rest.RESTRequest;
+
+/** Represents a request to create a policy. */
+@Getter
+@EqualsAndHashCode
+@ToString
+public class PolicyCreateRequest implements RESTRequest {
+
+  @JsonProperty("name")
+  private final String name;
+
+  @JsonProperty("comment")
+  @Nullable
+  private final String comment;
+
+  @JsonProperty("policyType")
+  private final String policyType;
+
+  @JsonProperty(value = "enabled", defaultValue = "true")
+  private final Boolean enabled;
+
+  @JsonProperty("content")
+  @JsonTypeInfo(
+      use = JsonTypeInfo.Id.NAME,
+      include = JsonTypeInfo.As.EXTERNAL_PROPERTY,
+      property = "policyType",
+      defaultImpl = PolicyContentDTO.CustomContentDTO.class)
+  @JsonSubTypes({
+    // add mappings for built-in types here
+    // For example: @JsonSubTypes.Type(value = DataCompactionContent.class, name =
+    // "system_data_compaction")
+  })
+  private final PolicyContentDTO policyContent;
+
+  @JsonProperty("exclusive")
+  private final Boolean exclusive;
+
+  @JsonProperty("inheritable")
+  private final Boolean inheritable;
+
+  @JsonProperty("supportedObjectTypes")
+  private final Set<MetadataObject.Type> supportedObjectTypes;
+
+  /**
+   * Creates a new PolicyCreateRequest.
+   *
+   * @param name The name of the policy.
+   * @param comment The comment of the policy.
+   * @param type The type of the policy.
+   * @param enabled Whether the policy is enabled.
+   * @param exclusive Whether the policy is exclusive.
+   * @param inheritable Whether the policy is inheritable.
+   * @param supportedObjectTypes The set of metadata object types that the policy can be applied to.
+   * @param content The content of the policy.
+   */
+  public PolicyCreateRequest(
+      String name,
+      String type,
+      String comment,
+      boolean enabled,
+      boolean exclusive,
+      boolean inheritable,
+      Set<MetadataObject.Type> supportedObjectTypes,
+      PolicyContentDTO content) {
+    this.name = name;
+    this.policyType = type;
+    this.comment = comment;
+    this.enabled = enabled;
+    this.exclusive = exclusive;
+    this.inheritable = inheritable;
+    this.supportedObjectTypes = supportedObjectTypes;
+    this.policyContent = content;
+  }
+
+  /** This is the constructor that is used by Jackson deserializer */
+  private PolicyCreateRequest() {
+    this.name = null;
+    this.policyType = null;
+    this.comment = null;
+    this.enabled = null;
+    this.exclusive = null;
+    this.inheritable = null;
+    this.supportedObjectTypes = null;
+    this.policyContent = null;
+  }
+
+  /**
+   * Validates the request.
+   *
+   * @throws IllegalArgumentException If the request is invalid, this exception is thrown.
+   */
+  @Override
+  public void validate() throws IllegalArgumentException {
+    Preconditions.checkArgument(
+        StringUtils.isNotBlank(name), "\"name\" is required and cannot be empty");
```

Comment:
You'd better unify the workd `can't be empty` is `null or empty` or just `empty`. I see you use `cannot be null or empty` in L287.

Reply:
where is the L287? the max number line is L170 in this file